spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/MqttPahoMessageDrivenChannelAdapter.java

```diff
@@ -375,11 +374,12 @@ public void messageArrived(String topic, MqttMessage mqttMessage) {
 		}
 	}
 
-	private @Nullable AbstractIntegrationMessageBuilder<?> toMessageBuilder(String topic, MqttMessage mqttMessage) {
+	@SuppressWarnings("NullAway") // MessageBuilder is not supposed to be null
```

Comment:
Well, actually, we also have just removed that ` Objects.requireNonNull`.
So, @SuppressWarnings("NullAway") is not only the reason for a null builder.
Let's just use a more generic comment: `// Dataflow analysis limitation` !